brian2/core/variables.py

```diff
@@ -296,17 +291,17 @@ class Constant(Variable):
     ----------
     name : str
         The name of the variable
-    unit : `Unit`
-        The unit of the variable. Note that the variable itself (as referenced
-        by value) should never have units attached.
+    dimensions : `Dimension`, optional
+        The dimensions of the variable. Note that the variable itself (as
```

Comment:
Again. I know that it just said dimensions before, but seems like a good opportunity to clean up potential source of confusion.